packages/experiments/src/components/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,153 @@
+    expect(slider.current!.value).toEqual(0);
+  });
+
+  it('should be able to handle zero value', () => {
+    const slider = React.createRef<ISlider>();
+
+    mount(<Slider label="slider" value={0} min={-100} max={100} componentRef={slider} />);
+    expect(slider.current!.value).toEqual(0);
+  });
+
+  it('renders correct aria-valuetext', () => {
+    let component = mount(<Slider />);
+
+    expect(component.find('.ms-Slider-slideBox').prop('aria-valuetext')).toBeUndefined();
+
+    const values = ['small', 'medium', 'large'];
+    const selected = 1;
+    const getTextValue = (value: number) => values[value];
+
+    component = mount(<Slider value={selected} ariaValueText={getTextValue} />);
+
+    expect(component.find('.ms-Slider-slideBox').prop('aria-valuetext')).toEqual(values[selected]);
+  });
+
+  it('formats the value when a format function is passed', () => {
+    const value = 10;
+    const valueFormat = (val: any) => `${val}%`;
+    const component = mount(<Slider value={value} min={0} max={100} showValue={true} valueFormat={valueFormat} />);
+    expect(component.find('label.ms-Label.ms-Slider-value').text()).toEqual(valueFormat(value));
+  });
+
+  it('calls onChanged after keyboard event', () => {
+    jest.useFakeTimers();
+    const onChanged = jest.fn();
+
+    const container = document.createElement('div');
+    document.body.appendChild(container);
+
+    ReactDOM.render(<Slider label="slider" defaultValue={12} min={0} max={100} onChanged={onChanged} />, container);
+    const sliderSlideBox = container.querySelector('.ms-Slider-slideBox') as HTMLElement;
+
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.up });
+    ReactTestUtils.Simulate.keyDown(sliderSlideBox, { which: KeyCodes.down });
+
+    expect(sliderSlideBox.getAttribute('aria-valuenow')).toEqual('9');
+
+    // onChanged should only be called after a delay
+    expect(onChanged).toHaveBeenCalledTimes(0);
+
+    setTimeout(() => {
+      expect(onChanged).toHaveBeenCalledTimes(1);
+    }, ONKEYDOWN_TIMEOUT_DURATION);
+
+    jest.runOnlyPendingTimers();
+
+    ReactDOM.unmountComponentAtNode(container);
+  });
+});
+
+it('should be able to display the correct number of custom labels & tickmarks', () => {
+  const labelsArray = [{ label: '20°C', value: 20 }, { label: '80°C', value: 80 }, { label: '100°C', value: 100 }];
+  const component = mount(<Slider marks={labelsArray} min={0} max={100} showValue={true} step={10} />);
+  const labelNumber = component.find('.ms-Slider-regularLabel').length;
+  expect(labelNumber).toEqual(3);
+  const tickNumber = component.find('.ms-Slider-regularTick').length;
+  expect(tickNumber).toEqual(11);
+});
+
+it('renders Slider with marks correctly', () => {
```

Comment:
Is this slider one with marks? The `<slider />` below does not have any props.. (but we don't really need a snapshot test for one with the props since examples are automatically snapshotted). May be ok to drop this test...